front/app/components/ContentUploadDisclaimer/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+// components
+import {
+  IconTooltip,
+  Text,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+
+// intl
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const ContentUploadDisclaimerTooltip = () => {
+  const { formatMessage } = useIntl();
+  const isMobileOrSmaller = useBreakpoint('phone');
+  const [showFullDisclaimer, setShowFullDisclaimer] = useState(false);
+
+  return (
+    <>
+      <IconTooltip
+        content={
+          <>
+            <Text m="0px" color="white" fontSize="s">
+              {formatMessage(messages.contentUploadDisclaimerTooltip)}
+            </Text>
+            <Text
+              onClick={(e) => {
+                e.preventDefault();
+                setShowFullDisclaimer(true);
+              }}
+              color="white"
+              fontSize="s"
+              m="0px"
+              mt="8px"
+              role="button"
+              style={{ cursor: 'pointer' }}
+              tabIndex={0}
+            >
+              <span style={{ textDecoration: 'underline' }}>
+                {formatMessage(messages.readFullDisclaimer)}
+              </span>
+              .
+            </Text>
+          </>
+        }
+        maxTooltipWidth={400}
+        placement={isMobileOrSmaller ? 'top' : 'auto'}
+      />
+      <Modal
+        opened={showFullDisclaimer}
+        close={() => {
+          setShowFullDisclaimer(false);
+        }}
+        closeOnClickOutside={true}
```

Comment:
I guess you're aware, but you can use shorthands when it's true.
```suggestion
        closeOnClickOutside
```